bin/docpub.js

```diff
@@ -10,18 +10,19 @@
 
 const program = require('commander');
 const pkg = require('../package.json');
-const DocpubPipeline = require('../lib/docpub');
+const Docpub = require('../lib/docpub');
 
 program
     .version(pkg.version)
-    .option('-v --verbose', 'verbose output')
-    .option('-p --path <path>', 'path to documents directory (defaults to process directory)');
+    .option('--doc-path <docPath>', 'path to documents directory (defaults to process directory)')
```

Comment:
I think this option should stay as `-p, --path` because we have already released a version with these CLI options. Also, this will probably be an often used option, so it should be easy to remember.

I understand that you changed it because there is now a `--config-path` option too, but I think that `--config-path` will be used much less often. Most often, the config file will be in the same directory as the repo.

What do you think?